src/runtime.rs

```diff
@@ -26,30 +26,37 @@ struct Stuff {
     outcome_error_fn: PyObject,
 }
 
-static STUFF: GILOnceCell<PyResult<Stuff>> = GILOnceCell::new();
+static STUFF: GILOnceCell<Stuff> = GILOnceCell::new();
 
-fn get_stuff(py: Python<'_>) -> PyResult<&Stuff> {
-    let res = STUFF.get_or_init(py, || {
-        let trio_lowlevel = py.import("trio")?.getattr("lowlevel")?;
-        let outcome = py.import("outcome")?;
-        Ok(Stuff {
-            tokio_runtime: tokio::runtime::Runtime::new()?,
-            trio_current_trio_token_fn: trio_lowlevel.getattr("current_trio_token")?.into_py(py),
-            trio_reschedule_fn: trio_lowlevel.getattr("reschedule")?.into_py(py),
-            trio_current_task_fn: trio_lowlevel.getattr("current_task")?.into_py(py),
-            trio_wait_task_rescheduled: trio_lowlevel.getattr("wait_task_rescheduled")?.into_py(py),
+fn get_stuff(py: Python<'_>) -> &Stuff {
+    STUFF.get_or_init(py, || {
+        let trio_lowlevel = py.import("trio").unwrap().getattr("lowlevel").unwrap();
+        let outcome = py.import("outcome").unwrap();
+        Stuff {
+            // It seems dangerous to use multithreaded tokio runtime where each thread call the python GIL
```

Comment:
1. Why ? The GIL isn't multi thread-safe ?
2. If find this comment need more context.